views/tools/applications.njk

```diff
@@ -0,0 +1,43 @@
+{% from "components/applicationSummary.njk" import readOnlySummary %}
+
+{% extends "layouts/tools.njk" %}
+{% block title %}Application{% endblock %}
+
+{% block content %}
+    <h1 class="display-4 mb-4">
+        {% if applicationData %}<a href="/tools/applications/{{ form.id }}">{% endif %}
+        {{ form.title }}
+        {% if applicationData %}</a>{% endif %}
+    </h1>
+    {% if applicationData %}
+        <h2><strong>Application reference</strong>: {{ applicationData.reference_id }}</h2>
+        <h3>
+            {{ formatDate(applicationData.createdAt, DATE_FORMATS.fullTimestamp) }}
+            <small title="{{ applicationData.createdAt }}">{{ timeFromNow(applicationData.createdAt) }}</small>
+        </h3>
+        <div class="card">
+            <div class="card-body">
+                {{ readOnlySummary(applicationData.application_data) }}
+            </div>
+        </div>
+    {% else %}
+        <h2>All applications ({{ applications.length }})</h2>
+        <p>Click an application below to view all its data.</p>
+        <div class="border mb-5">
+            <div class="list-group">
+                {% for application in applications %}
+                    <a class="list-group-item list-group-item-action flex-column align-items-start"
+                       href="/tools/applications/{{ form.id }}/{{ application.reference_id }}">
```

Comment:
Are we able to avoid hard-coding the route here? Does a relative link work e.g. `./{{ form.id }}/{{ application.reference_id }}`